addons/account/models/account_bank_statement.py

```diff
@@ -828,6 +834,17 @@ def _compute_is_reconciled(self):
                 # The journal entry seems reconciled.
                 st_line.is_reconciled = True
 
+            # Compute residual amount
+            st_line_currency = suspense_lines.currency_id or suspense_lines.company_currency_id
+            balance_field, residual_field = ('amount_currency', 'amount_residual_currency') if suspense_lines.currency_id else ('balance', 'amount_residual')
+
+            if self.to_check:
+                st_line.amount_residual = -sum(liquidity_lines.mapped(balance_field))
+            elif suspense_lines.account_id.reconcile:
+                st_line.amount_residual = sum(suspense_lines.mapped(residual_field))
+            else:
+                st_line.amount_residual = sum(suspense_lines.mapped(balance_field))
```

Comment:
I don't think the sign is correct. A statement line of 100 will have a residual of -100. Is it expected? Seems yes because OCO told me "fin de l'histoire, rideau"